### PR TITLE
Partial matching for layout headers

### PR DESCRIPTION
**Pull request by PatrykSzwer**
For #118

**Needs https://github.com/StarcounterApps/Simplified/pull/110 to work.**

### CODE REVIEW

File: src/BlendingProvider/Website/Api/ContentHandlers.cs [hunk 1]
```diff
@@ -355,7 +355,31 @@ private bool FindUrlByHeaders(WebUrl webUrl, Dictionary<string, string> requestH
             {
                 return true;
             }
-            return webUrl.Headers.All(wh => requestHeaders.Any(rh => rh.Key.Equals(wh.Name, StringComparison.InvariantCultureIgnoreCase) && rh.Value == wh.Value));
+
+            if (!webUrl.Headers.Any(x => x.PartialMatching))
```
Comment (un-tone) [inline src/BlendingProvider/Website/Api/ContentHandlers.cs]: This statement looks redundant. The same is doing below.

Comment (PatrykSzwer) [reply src/BlendingProvider/Website/Api/ContentHandlers.cs]: TBH I added this line there, to save some performance, but now when I think of it, probably version with partial matching will be even more (or at least same amount of time) used as an old one.
I'll remove this one.

**Comment by kegor** [comment]
BlendingEditor is runnable locally with the change in Simplified, but at night I only can't quickly imagine the test-case for checking the exact functionality, @PatrykSzwer could you please give me a hint?

**Comment by PatrykSzwer** [comment]
You can just add header rule to topbarLayout, lets say host: localhost and check how its working with/without partial matching.

But TBH I shouldn't be the one to suggest you how you should test it, because you could find new bug that I may not find trough my testing process.

**Comment by PatrykSzwer** [comment]
I updated package with simplified. After tests pass I'll merge.

**Comment by PatrykSzwer** [comment]
There are some tests that failed but I talked to @kegor and those are not related to my changes (new flaky tests). I'll merge this and #106 should be reopened.

**Pull request merged**